.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: python
 
 python:
-    - "2.7"
     - "3.5"
     - "3.6"
+    - "3.8"
```

Comment:
Don't you also need to add 3.7 here as well?

Reply:
So when I added it, Travis actually duplicated it. I'm not 100% sure why.

Reply:
Actually let me remove the custom setting from the files and just rely on Travis to do it...

Reply:
I think that is because of the matrix include below.

Reply:
Yeah, that's exactly what I'm thinking. That was an old workaround for Travis not launching 3.7 -- might be fixed now.